packages/@cdktf/provider-generator/lib/get/generator/emitter/attributes-emitter.ts

```diff
@@ -156,6 +159,14 @@ export class AttributesEmitter {
     }
 
     this.code.line(`// Getting the computed value is not yet implemented`);
+    if (type.isSet) {
+      // Token.asAny is required because tolist returns an Array encoded token which the listMapper
+      // would try to map over when this is passed to another resource. With Token.asAny() it is left
+      // as is by the listMapper and is later properly resolved to a reference
+      // (this only works in TypeScript currently, same as for referencing lists
+      // [see "interpolationForAttribute(...) as any" further below])
```

Comment:
If we don't get to replace complex computed lists with native lists for 0.9 we might need to add sth to the docs around set access only supported in TS, don't we?

Reply:
Hm, that's the same as it already is. Simple sets (i.e. accessing a String set with `element()`) should work now without casting the set to a list using `tolist()` in any language.